NCPCore/src/main/java/fr/neatmonster/nocheatplus/utilities/map/BlockProperties.java

```diff
@@ -2814,6 +2814,30 @@ public static final boolean isAir(Material type) {
                 ;
     }
 
+    /**
+     * Convenience method including null check.
+     *
+     * @param type
+     *            the type
+     * @return true, if is Scaffolding
+     */
+    public static final boolean isScaffolding(Material type) {
+
+        return type != null && type == Material.SCAFFOLDING;
```

Comment:
Will cause NoSuchFieldError SCAFFOLDING on a legacy version like 1.12. Should use BridgeMaterial, do the same like void_air!